src/components/Review/Review.js

```diff
@@ -0,0 +1,34 @@
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import {Avatar, Paper, Rating, Stack} from "@mui/material";
+import classes from "pages/BookDetails/BookDetails.module.scss";
+import StarIcon from "@mui/icons-material/Star";
+
+const Review = (props) => {
```

Comment:
Hajde ovde u zagradama da pisemo props. To bi izgledalo ovako: ({ prop1, prop2, prop3,... })